<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>display_config_seite</class>
 <widget class="QWidget" name="display_config_seite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Display-Konfiguration</string>
  </property>
  <property name="styleSheet">
   <string></string>
  </property>

  <!-- HEADER -->
  <widget class="QLabel" name="lbl_header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string></string>
   </property>
   <property name="text">
    <string>Display-Konfiguration</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>

  <!-- BACK BUTTON -->
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string></string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>

  <!-- HELLIGKEIT GRUPPE -->
  <widget class="QLabel" name="lbl_helligkeit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Display-Helligkeit:</string>
   </property>
  </widget>

  <widget class="QSlider" name="slider_helligkeit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>31</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>5</number>
   </property>
  </widget>

  <widget class="QLabel" name="lbl_helligkeit_wert">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string></string>
   </property>
   <property name="text">
    <string>48%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>

  <!-- THEME GRUPPE -->
  <widget class="QLabel" name="lbl_theme">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Farbschema:</string>
   </property>
  </widget>

  <widget class="QComboBox" name="combo_theme">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QComboBox {
    background-color: white;
    
    border-radius: 8px;
    padding: 8px;
}
QComboBox:drop-down {
    border: none;
}
QComboBox::down-arrow {
    width: 12px;
    height: 12px;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Standard</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nacht (Blau)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Natur (Gr√ºn)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ultra-Dunkel</string>
    </property>
   </item>
  </widget>

  <!-- AUTO NACHT-MODUS -->
  <widget class="QCheckBox" name="chk_auto_nachtmodus">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Nacht-Modus (22:00 - 06:00 Uhr)</string>
   </property>
  </widget>

  <!-- VORSCHAU BEREICH -->
  <widget class="QLabel" name="lbl_vorschau">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>150</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Vorschau:</string>
   </property>
  </widget>

  <widget class="QFrame" name="frame_vorschau">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>220</y>
     <width>450</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QFrame {
    
    
    border-radius: 15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   
   <!-- Vorschau-Inhalte -->
   <widget class="QLabel" name="lbl_vorschau_titel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>410</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>  padding: 5px</string>
    </property>
    <property name="text">
     <string>üê¥ Futterkarre Vorschau</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>

   <widget class="QLabel" name="lbl_vorschau_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>410</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Gewicht: 45.2 kg
Pferd: Beispiel-Pferd
Status: Bereit zum F√ºttern</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignTop</set>
    </property>
   </widget>

   <widget class="QPushButton" name="btn_vorschau_demo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
   <property name="styleSheet">
    <string>QPushButton {
    
    color: white;
    padding: 5px;
}</string>
   </property>
    <property name="text">
     <string>Demo Button</string>
    </property>
   </widget>
  </widget>

  <!-- SPEICHERN BUTTON -->
  <widget class="QPushButton" name="btn_speichern">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>550</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton {
    
    color: white;
}</string>
   </property>
   <property name="text">
    <string>üíæ Speichern</string>
   </property>
  </widget>

  <!-- RESET BUTTON -->
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>550</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton {
    
    color: white;
}</string>
   </property>
   <property name="text">
    <string>üîÑ Zur√ºcksetzen</string>
   </property>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>